src/TStypes.tsx

```diff
@@ -0,0 +1,20 @@
+
+export interface CardsDataDTO {
+    title?: string;
+    oldTitle?: string;
+    body?: string;
+    oldDescription?: string;
+    id: string ;
+}
+export interface ReducerState {
+    data:CardsDataDTO[],
+    isEdit: boolean,
+    isOpen: boolean,
+    isLoad: boolean,
+    }
+
+export type Action =
```

Comment:
I would recommend to move this types to the redux folder to the file `actionTypes`.

Reply:
done